### ISSUE TITLE
Client crashes

### ISSUE THREAD

**Issue by zefir-git** [plasmoapp/plasmo-voice#431]
Client version 2.0.10 and 2.0.8 (both Fabric) crashes with:
```
[21:50:12] [Netty Client IO #3/INFO]: Connecting to n1.de.net.cloudnode.pro.:50008
[ALSOFT] (EE) Failed to set real-time priority for thread: Operation not permitted (1)
[21:50:13] [Netty Client IO #3/INFO]: Device Hesh ANC initialized
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGFPE (0x8) at pc=0x00007f3d808db891, pid=263030, tid=263291
#
# JRE version: OpenJDK Runtime Environment Corretto-17.0.7.7.1 (17.0.7+7) (build 17.0.7+7-LTS)
# Java VM: OpenJDK 64-Bit Server VM Corretto-17.0.7.7.1 (17.0.7+7-LTS, mixed mode, sharing, tiered, compressed oops, compressed class ptrs, g1 gc, linux-amd64)
# Problematic frame:
# C  [jna10557168939388225295.tmp+0x40891]  opus_encoder_ctl+0xf91
#
# Core dump will be written. Default location: Core dumps may be processed with "/usr/lib/systemd/systemd-coredump %P %u %g %s %t %c %h" (or dumping to /home/zefir/.local/share/PrismLauncher/instances/1.20.1/.minecraft/core.263030)
#
# An error report file with more information is saved as:
# /home/zefir/.local/share/PrismLauncher/instances/1.20.1/.minecraft/hs_err_pid263030.log
#
# If you would like to submit a bug report, please visit:
#   https://github.com/corretto/corretto-17/issues/
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#
Process crashed with exitcode 6.
```

**Comment by apehum** [comment]
Try using 2.1.x alpha version: https://modrinth.com/plugin/plasmo-voice/versions?c=alpha&l=fabric

**Comment by zefir-git** [comment]
> Try using 2.1.x alpha version: https://modrinth.com/plugin/plasmo-voice/versions?c=alpha&l=fabric

Not available for 1.20.6

**Comment by apehum** [comment]
In that case, the only possible solution is to disable natives by adding java argument: `-Dplasmovoice.disable_natives=true`.
